packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -149,3 +152,24 @@ export function wrapSummaryInChannelsTree(summarizeResult: ISummaryTreeWithStats
     };
     summarizeResult.stats.treeNodeCount++;
 }
+
+export async function getFluidDataStoreAttributesFromStorageOrSnapshot(
+    storage: IDocumentStorageService,
+    snapshot: ISnapshotTree,
+): Promise<ReadFluidDataStoreAttributes> {
+    // Need to rip through snapshot.
```

Comment:
I'm not sure "rip" is the right word (here and in other places). Consider using something else

Reply:
Ok :) I copied comment from the other place.